portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/questions/QuestionsEntry.testcase

```diff
@@ -40,79 +40,93 @@ definition {
 		}
 	}
 
-	@description = "Validate if the user can create a question with an AL image."
+	@description = "LPS-110392 - Verifies if the user can create a question with an AL image."
```

Comment:
LPS-110392 - Verify if the user can create a question with an image from a connected Depot